checks.d/activemq_xml.py

```diff
@@ -42,27 +42,38 @@ def check(self, instance):
         detailed_queues = instance.get("detailed_queues", [])
         detailed_topics = instance.get("detailed_topics", [])
         detailed_subscribers = instance.get("detailed_subscribers", [])
+        suppress_errors = bool(instance.get("suppress_errors", None))
```

Comment:
You should use the `is_affirmative` method https://github.com/DataDog/dd-agent/blob/7a401319c6318eb63f16e68580cfacda9b373bb7/config.py#L212-L217